hub/ui/src/components/search-bar/SearchBar.tsx

```diff
@@ -36,8 +36,11 @@ const SearchBar: React.FC = (props: any) => {
   const onTextChanged = (e: any) => {
     const value = e;
     let suggestions: any = [];
-    const regex = new RegExp(`${value}`, 'i');
-    suggestions = props.TaskDataList.sort().filter((v: any) => regex.test(v.name));
+    const nameregex = /-/gi;
+    const regex = new RegExp(`${ value.replace(nameregex, ' ') }`, 'i');
+    suggestions = props.TaskDataList.sort().filter((v: any) =>
```

Comment:
this fails when the search term has leading or trailing spaces. try: `tower cli    ` e.g.

Reply:
yes its failing :cry: